stix2/sources/filesystem.py

```diff
@@ -27,87 +42,181 @@ def __init__(self, stix_dir="stix_data"):
 
 
 class FileSystemSink(DataSink):
-    """
+    """FileSystemSink
+
+    Provides an interface for adding/pushing STIX objects
+    to file directory of STIX objects.
+
+    Can be paired with a FileSystemSource, together as the two
+    components of a FileSystemStore.
+
+    Args:
+        stix_dir (str): path to directory of STIX objects
+
     """
     def __init__(self, stix_dir="stix_data"):
```

Comment:
Does it make sense to have a default value for this kwarg? I know it isn't part of this change, but I just noticed it.

Reply:
Nah, that was just there from the beginning.